weblate/trans/models/label.py

```diff
@@ -46,10 +46,8 @@ class Meta:
         verbose_name_plural = "label"
 
     def __str__(self):
-        return mark_safe(
-            '<span class="label label-{}">{}</span>'.format(
-                self.color, escape(self.name)
-            )
+        return format_html(
+            '<span class="label label-{}">{}</span>', mark_safe(self.color), self.name
```

Comment:
```suggestion
            '<span class="label label-{}">{}</span>', self.color, self.name
```

I'd omit `mark_safe` here even though the color list is currently hard-coded (see https://github.com/WeblateOrg/weblate/blob/main/weblate/utils/colors.py).